src/components/CanvasContainer/CanvasContainer.tsx

```diff
@@ -1,13 +1,14 @@
 import React from "react";
-import { canvasElement } from "../../canvas/Canvas";
+import { renderer } from "../../canvas/Renderer";
 import './CanvasContainer.css';
 
 const CanvasContainer: React.FC = () => {
   const canvasContainerRef = React.useRef(document.createElement("div"));
+  const canvas = renderer.init();
 
   React.useEffect(() => {
-    canvasContainerRef.current.appendChild(canvasElement);
-  }, []);
+    canvasContainerRef.current.appendChild(canvas);
+  });
```

Comment:
the reason I added the empty array as an argument is because of the note in the yellow box on [the documentation](https://reactjs.org/docs/hooks-reference.html#useeffect). Basically is makes sure the canvas will only be mounted once on its container. But yeah, I did test it with or without the parameter and could not find a difference on how it works in the browser, so we could leave it out.

Reply:
I removed it because the linter suggested it. Based on the documentation, I think you are right that we should keep it. I'll add it back.

Reply:
yeah, I added something like that in the addPhoto branch

Reply:
It will change as we go, but for now what you propose sounds good